keyboard/android/src/main/java/com/capacitorjs/plugins/keyboard/Keyboard.java

```diff
@@ -84,7 +89,11 @@ public void onGlobalLayout() {
                     int resultBottom = r.bottom;
                     int screenHeight;
 
-                    if (Build.VERSION.SDK_INT >= 23) {
+                    if (Build.VERSION.SDK_INT >= 30) {
+                        Insets windowInsets = rootView.getRootWindowInsets().getInsetsIgnoringVisibility(WindowInsets.Type.systemBars());
+                        screenHeight = rootViewHeight;
+                        resultBottom = windowInsets.bottom;
+                    } else if (Build.VERSION.SDK_INT >= 23) {
```

Comment:
```suggestion
                    } else if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
```
use the constant instead of the hardcoded value